src/app/public/modules/action-button/types/action-button-container-align-items.ts

```diff
@@ -0,0 +1,13 @@
+export enum SkyActionButtonContainerAlignItems {
+
+  /**
+   * Displays the action buttons as center-aligned.
+   */
+  center = 'center',
```

Comment:
In the past we've capitalized the enum's key; possible to do that here?
```
Center = 'center'
```

Reply:
Done.